### PR TITLE
Added noetic dockerfiles for ubuntu focal

### PR DESCRIPTION
**Pull request by ahcorde**
we need to create the noetic dockerfiles to support some CI jobs (for example in the perception pipeline: https://github.com/ros-perception/image_pipeline/pull/507).

Meta packages such as: `ros-noetic-ros-core`, `ros-noetic-ros-base`, etc are not created yet. Fow now it's not possible to build these dockerfiles.

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: ros/noetic/ubuntu/focal/robot/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# This is an auto generated Dockerfile for ros:robot
+# generated from docker_images/create_ros_image.Dockerfile.em
+FROM ros:noetic-ros-base-focal
+
+# install ros packages
+RUN apt-get update && apt-get install -y \
+    ros-focal-robot \
```
Comment (wxmerkt) [inline ros/noetic/ubuntu/focal/robot/Dockerfile:7]: I believe this should be `ros-noetic-robot`

File: ros/noetic/ubuntu/focal/ros-core/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    dirmngr \
+    gnupg2 \
+    && rm -rf /var/lib/apt/lists/*
+
+# setup keys
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
+
+# setup sources.list
+RUN echo "deb http://repositories.ros.org/ubuntu/testing focal main" > /etc/apt/sources.list.d/ros1-latest.list
+RUN echo "deb http://packages.ros.org/ros/ubuntu bionic main" > /etc/apt/sources.list.d/ros1-latest.list
```
Comment (wxmerkt) [inline ros/noetic/ubuntu/focal/ros-core/Dockerfile:21]: `focal main`?

**Comment by mikaelarguedas** [comment]
@wxmerkt 

> Is there a "working" Dockerfile for debugging jobs on the noetic buildfarm yet?

Recommended way to debug buildfarm jobs is to use the ros_buildfarm package.
Some information on how to run buildfarm jobs locally can be found at https://github.com/ros-infrastructure/ros_buildfarm/blob/master/doc/index.rst#run-jobs-locally

---

@ahcorde the docker_images on this repositories are generated. However they won't be generated and pushed upstream before the release actually happens.
As this PR looks hand-crafted and not complying with the templates or expected tags, it is likely that it will be rejected in favor of a PR containing the generated Dockerfiles

**Comment by ahcorde** [comment]
Sure, closed

**Comment by mikaelarguedas** [comment]
@ahcorde Are you using this branch for something on your side? or can it be safely deleted ?

**Comment by wxmerkt** [comment]
@mikaelarguedas Thank you for the pointers, much appreciated :-)

**Comment by ahcorde** [comment]
Removed @mikaelarguedas

**Pull request closed without merging**